doc/tutorial.rst

```diff
@@ -21,132 +25,159 @@ also the :doc:`primer` which focuses on a particular problem in detail.
 Users not familiar with the finite element method should start with the
 :ref:`sec-solving-pdes-fem`.
 
-Basic notions
+.. _invoking_from_command_line:
+
+Invoking *SfePy* from the Command Line
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This section introduces the basics of running *SfePy* from the command line.
+The ``$`` indicates the command prompt of your terminal.
+
+* The script `simple.py` is the most basic starting point in *SfePy*. It
+  can be invoked either directly::
+
+    $ ./simple.py <problem_description_file.py>
```

Comment:
- this example is used a couple of lines bellow ("Running a simulation")
- this part of doc should/will be re-written anyway...